.github/pull_request_template.md

```diff
@@ -1,23 +1,25 @@
+<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
 ## Summary of the Pull Request
 
-**What is this about:**
+<!-- Please review the items on the PR checklist before submitting-->
+## PR Checklist
 
-**What is included in the PR:** 
-
-**How does someone test / validate:** 
-
-## Quality Checklist
-
-- [ ] **Linked issue:** #xxx
-- [ ] **Communication:** I've discussed this with core contributors in the issue. 
+- [ ] **Closes:** #xxx
+- [ ] **Communication:** I've discussed this with core contributors already. If work hasn't been agreed, this work might be rejected
 - [ ] **Tests:** Added/updated and all pass
-- [ ] **Installer:** Added/updated and all pass
 - [ ] **Localization:** All end user facing strings can be localized
-- [ ] **Docs:** Added/ updated
-- [ ] **Binaries:** Any new files are added to WXS / YML
-   - [ ] No new binaries
-   - [ ] [YML for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/pipeline.user.windows.yml#L68) for new binaries
-   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries
+- [ ] **Dev docs:** Added/updated
+- [ ] **New binaries:** Added on the required places
+   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
+   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
+   - [ ] [YML for CI pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects
+   - [ ] [YML for signed pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/release.yml)
+- [ ] **Documentation updated:** If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) and link it here: #xxx
+- [ ] **CLA signed:** A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
```

Comment:
Does this make sense. The cla agreement is validated automatically and regardless of the check we then have to check this box/item. I see no usecase for this checkbox. (Especially as we have the requirement check for this where we can see the state in real time.)

I think this should be only a note as before/in our productive template.

_(Sory for not mentioning this in my first review.)_